src/Component.test.js

```diff
@@ -0,0 +1,13 @@
+import { $, expect } from '@wdio/globals'
```

Comment:
Do we need this test?

Reply:
It was installed with wdio. I'd like to keep it to be as an example)